resources/tickets.py

```diff
@@ -88,7 +88,14 @@ def get(self):
     @jwt_required
     def post(self):
         data = Tickets.parser.parse_args()
-        ticket = TicketModel(**data)
+        ticket = TicketModel(
+            issue=data.issue,
+            tenantID=data.tenant,
```

Comment:
Lets get an issue created for the FE to update the parameter names they're sending to us.

Reply:
I'll create an issue and link it to this PR~